content/2018-munich/schedule.xml

```diff
@@ -467,13 +480,14 @@ we will have a write-in poster for people interested in giving a LT.
     <room name="Main room">
       <event id="22">
         <start>9:30</start>
-        <duration>0:30</duration>
+        <duration>00:30</duration>
         <room>Main room</room>
         <abstract/>
         <title>Improving Reliability Through Engineering an Easy-to-use Prometheus-Based Monitoring and Alerting Stack: Introducing Our Reliability Toolkit</title>
         <type>talk</type>
         <released>True</released>
         <persons>
+          <person>Robin van Zijll</person>
           <person>Janna Brummel (ING N.V.)</person>
```

Comment:
Janna has been removed as a speaker; where did your script pull that info?

Reply:
From `content/2018-munich/schedule.md`, Janna is still listed there